### DISCUSSION TITLE
Move Prover stuck at `configured prover "" could not be found`

### DISCUSSION THREAD

**Discussion by SteMak** [aptos-labs/aptos-developer-discussions#140] [Questions] [answered]
Trying to run `aptos move prove` but getting stuck with the error during the `running solver` stage
> "Error": "Move Prover failed: The configured prover `` could not be found"

I suppose some deps are missed but had not succeed in my search.

**Comment by Alexzander-Stone** [comment, marked as answer]
Hi @SteMak, what steps did you take to set up the Move Prover? Additionally, please include your OS.

In the [installation guide](https://aptos.dev/tools/aptos-cli/install-cli/install-move-prover), you'll notice that you need to specify that you're installing additional Move tools. 
  For example, Linux/macOS users will use `./scripts/dev_setup.sh -yp`, where `-yp` is required to install Move Prover dependencies.

**Comment by SteMak** [comment]
Thank you for your answer
Yeap, there were missed dependencies successfully installed by the `dev_setup` script

Be aware, that the script may additionally upgrade some system components
